### ISSUE TITLE
List of players in "Transfer queue to another player" should be sorted

### ISSUE THREAD

**Issue by madbrain76** [music-assistant/hass-music-assistant#2899]
### What version of Music Assistant has the issue?

2.3.0b19

### What version of the Home Assistant Integration have you got installed?

None

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

When using "Transfer queue to another player", the list of players appears in a seemingly random orders 

![image](https://github.com/user-attachments/assets/92797ed0-818b-4c73-b9d6-ea5e41713bfb)

### How to reproduce

1. set up a player provider with multiple players
2. play content on one of the players
3. open the queue
4. on the top right, select "Transfer queue to another player"
5. 

### Music Providers

Qobuz, but this is independent of music provider

### Player Providers

Chromecast, but this is independent of player provider

### Full log output

[ma_transfer.log](https://github.com/user-attachments/files/16970151/ma_transfer.log)

### What version of Home Assistant Core are your running

2024.9.1

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Windows

**Comment by marcelveldt** [comment]
should be fixed in 3.2.0b22

**Comment by madbrain76** [comment]
Looks like it's mostly fixed in b23. There is still one oddity. The entry at the top that says "guest bedroom AVR" isn't in the right spot.

![Screenshot_20240914_024813_Home Assistant](https://github.com/user-attachments/assets/2aa005b8-cb1d-4f04-982a-8101b63bcbb4)

That device is the AirPlay version of the Marantz NR1603 AVR. The AVR is powered off. It's DLNA device is showing as being off. The Airplay probably should, too. That may be a separate issue. But it still probably shouldn't show at the top of the list.

![Screenshot_20240914_024747_Home Assistant](https://github.com/user-attachments/assets/ade49024-9ead-48ea-a9fe-27d4b1ed90a2)

**Comment by OzGav** [comment]
The fact that it is showing at the top of the list in the settings as well suggests to me that something else other than the original report is at play here. What if you rename the player?

**Comment by madbrain76** [comment]
Renaming did the trick,  but only after I killed/restarted the HA Android app. Is there a cleaner way to force a refresh?

![Screenshot_20240914_030453_Home Assistant](https://github.com/user-attachments/assets/30eb3809-3924-419a-b73c-2a31d09b00f6)

**Comment by OzGav** [comment]
Probably needed to flush the browser cache?
